packages/cxl-ui/scss/cxl-certificate-header.scss

```diff
@@ -0,0 +1,111 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+@use "./mixins";
+
+:host {
+  color: var(--lumo-tint);
+  background-color: var(--lumo-shade);
+  @include mixins.pesudo-element-full-width(var(--lumo-shade));
+
+  .container {
+    position: relative;
+    box-sizing: border-box;
+    width: 100%;
+    max-width: var(--cxl-content-max-width-wide);
+    padding: var(--lumo-space-l) 0;
+
+    @media #{mq.$small} {
+      display: flex;
+      justify-content: space-between;
+      gap: var(--lumo-space-m);
+    }
+
+    @media #{mq.$large} {
+      padding: var(--lumo-space-xl) 0;
+    }
+  }
+
+  .content {
+    max-width: var(--cxl-hero-content-max-width);
+  }
+
+  .title {
+    margin: 0;
+    font-family: var(--cxl-lumo-font-secondary);
+  }
+
+  .completed {
+    display: flex;
+    margin-top: var(--lumo-space-l);
+    font-size: var(--lumo-font-size-xl);
+    font-weight: 700;
+    color: var(--lumo-primary-color);
+  }
+
+  .completed-icon {
+    display: inline-flex;
+    width: 32px;
+    height: 32px;
+    align-items: center;
+    justify-content: center;
+    margin-right: var(--lumo-space-m);
+    color: var(--lumo-primary-color);
+    background-color: var(--lumo-primary-color-10pct);
+    border-radius: 100%;
+  }
+
+  .description {
+    margin-top: var(--lumo-space-l);
+    font-family: var(--cxl-lumo-font-secondary);
+    line-height: var(--lumo-line-height-m);
+  }
+
+  .stats {
+    margin-top: var(--lumo-space-m);
+
+    @media #{mq.$medium} {
+      margin-top: calc(4 * var(--lumo-space-m));
+    }
+  }
+
+  .actions {
+    display: flex;
+    flex-direction: column;
+    gap: var(--lumo-space-s);
+    margin-top: var(--lumo-space-m);
+
+    @media #{mq.$medium} {
+      flex-direction: row;
+      gap: var(--lumo-space-m);
+      margin-top: var(--lumo-space-xl);
+    }
+  }
+
+  .credential {
+    width: 100%;
+    max-width: 100%;
+    margin-top: var(--lumo-space-l);
+    padding-top: var(--lumo-space-l);
+    border-top: solid 1px var(--cxl-color-dark-gray);
+
+    @media #{mq.$medium} {
+      max-width: 483px;
```

Comment:
Where does it come from?

Reply:
@pawelkmpt From design. Added a variable `--cxl-hero-certificate-max-width` and rounded the value to 480px.

In the future, we could consider a grid-based approach. The entire web would align within a grid. In such a case, each element is always a certain number of columns wide, and there is no need to think about max width values. And we don't need to know how wide one column is. The number of columns can be adjusted at breakpoints.
cc @freudFlintstone 

Example (certificate is 5 columns wide on desktop and full width on mobile):
![Screenshot 2023-11-29 at 12 30 22](https://github.com/conversionxl/aybolit/assets/70375089/2f0a8edc-d6e2-40d5-b88f-4fa89073f7c0)